.styleguidist/styleguidist.sections.js

```diff
@@ -130,6 +130,8 @@ module.exports = {
             'SkipLink/SkipLink',
             'ExternalLink/ExternalLink',
             'RouterLink/RouterLink',
+            'ListLink/ListLink',
```

Comment:
LinkList could have it's own page, I found it a bit hard to find among links, and /Components/Link page is quite long

Reply:
^ Agreed

Reply:
It now has its own page on the top level.